Source/Scene/Cesium3DTile.js

```diff
@@ -935,6 +936,21 @@ function createPriorityFunction(tile) {
   };
 }
 
+function getJsonContent(arrayBuffer, byteOffset) {
```

Comment:
Another thing, this function seems very general-purpose, does it make sense to put this in Core since this is just an extended version of `getStringFromTypedArray`?

Reply:
I'll open a PR into `master` that adds `getJsonFromTypedArray`